code/modules/reagents/oldchem/reagents/_reagent_base.dm

```diff
@@ -63,13 +63,18 @@
 	return
 
 /datum/reagent/proc/on_mob_life(var/mob/living/M as mob, var/alien)
-	if(!istype(M, /mob/living)) // YOU'RE A FUCKING RETARD NEO WHY CAN'T YOU JUST FIX THE PROBLEM ON THE REAGENT - Iamgoofball
-		return //Noticed runtime errors from facid trying to damage ghosts, this should fix. --NEO
-				// Certain elements in too large amounts cause side-effects
+	if(!istype(M))
+		return
+	if(reagent_state == GAS && M.stat != DEAD && ishuman(M))
+		// make gas reagents affect people like the gasses
+		var/mob/living/carbon/human/H = M
+		if(id == H.species.breath_type)
+			M.adjustOxyLoss(-2*REM)
```

Comment:
I'd rather this just go away, to be honest; cramming oxygen pills down people's throats to clear up any oxygen damage is kinda lame, IMO.